### PR TITLE
[AppBar] Adopt safe area layout guide in MDCAppBarContainerViewController.

### PR DESCRIPTION
**Pull request by Arcank**

### CODE REVIEW

File: components/AppBar/src/MDCAppBarContainerViewController.m [hunk 1]
```diff
@@ -44,6 +44,31 @@ - (void)viewDidLoad {
 
   [_appBar addSubviewsToParent];
 
+   UIView *contentView = self.contentViewController.view;
```
Comment (randallli) [inline components/AppBar/src/MDCAppBarContainerViewController.m]: indent nit: delete one space.

**Review by randallli — APPROVED** [review]

**Comment by ianegordon** [comment]
Blocked pending a test against internal clients.

**Comment by Arcank** [comment]
Actually, MDCUnitTests seem to have failed on iOS 8 https://travis-ci.org/material-components/material-components-ios/jobs/336199057

I will need to fix.

**Comment by ianegordon** [comment]
Hi @Arcank !  Sorry for the delayed response, we were in a code freeze last week.  Are you still planning on updating with the iOS 8 fix?

**Comment by Tdabai** [comment]
我需要定制产品 qq2037338069

**Comment by Tdabai** [comment]
希望与我联系 2037338069

**Comment by jverkoey** [comment]
Closing this due to inactivity. Please reopen if still desired.

**Pull request closed without merging**